extension/src/popup/components/accountMigration/ReviewMigration/index.tsx

```diff
@@ -0,0 +1,248 @@
+
+        // eslint-disable-next-line no-await-in-loop
+        const { account, isSigner } = await getAccountInfo({
+          publicKey,
+          networkDetails,
+        });
+
+        const DEFAULT_ACCT_ITEM = {
+          publicKey,
+          name: migratableAccounts[i].name,
+          trustlines: 0,
+          dataEntries: 0,
+          xlmBalance: "",
+          isSigner,
+          minBalance: "",
+        };
+
+        let acctItem = {
+          ...DEFAULT_ACCT_ITEM,
+        };
+
+        if (account) {
+          const minBalance = new BigNumber(
+            (2 + account.subentry_count) * BASE_RESERVE,
+          ).toString();
+
+          acctItem = {
+            ...DEFAULT_ACCT_ITEM,
+            trustlines: account.balances.length - 1,
+            dataEntries: Object.keys(account.data_attr).length,
+            xlmBalance: account.balances[account.balances.length - 1].balance,
+            minBalance,
+          };
+        }
+
+        acctItemArr.push(acctItem);
+      }
+
+      setAccountList(acctItemArr);
+    };
+
+    fetchAccountData();
+  }, [networkDetails]);
+
+  const handleSubmit = (values: FormValues) => {
+    dispatch(saveIsMergeSelected(values.isMergeSelected));
+    navigateTo(ROUTES.accountMigrationMnemonicPhrase);
+  };
+
+  const initialValues: FormValues = {
+    isMergeSelected: false,
+  };
+
+  const ReviewMigrationFormSchema = YupObject().shape({
+    isMergeSelected: YupBoolean(),
+  });
+
+  return (
+    <div className="ReviewMigration">
+      <header className="ReviewMigration__header">
+        <MigrationHeader>{t("Review accounts to migrate")}</MigrationHeader>
+        <MigrationParagraph>
+          {t("Only accounts ready for migration will be migrated.")}
+        </MigrationParagraph>
+      </header>
+      {accountList.length ? (
+        <AccountListItems accountList={accountList} />
+      ) : (
+        <div className="ReviewMigration__loader">
+          <Loader />
+        </div>
+      )}
+      <Formik
+        onSubmit={handleSubmit}
+        initialValues={initialValues}
+        validationSchema={ReviewMigrationFormSchema}
+      >
+        {({ isSubmitting }) => (
+          <Form className="NetworkForm__form">
+            <div className="ReviewMigration__option">
+              <Field name="isMergeSelected">
+                {({ field }: FieldProps) => (
+                  <Checkbox
+                    fieldSize="md"
+                    autoComplete="off"
+                    id="isMergeSelected-input"
+                    label={
+                      <div>
+                        <span className="ReviewMigration__highlight">
+                          {t("Optional")}:{" "}
+                        </span>
+                        {t(
+                          "Merge accounts after migrating (you funding lumens used to fund the current accounts will be sent to the new ones - you lose access to the current accounts.)",
```

Comment:
typo? `you funding lumens` => `your funding lumens`

Reply:
ah good catch!